R-package/README.md

```diff
@@ -1,5 +1,4 @@
 [![CRAN Version](https://r-pkg.org/badges/version/RGF)](https://cran.r-project.org/package=RGF)
-[![codecov.io](https://codecov.io/github/RGF-team/rgf/coverage.svg?branch=master)](https://codecov.io/github/RGF-team/rgf?branch=master)
```

Comment:
This badge cause the following error at Appveyor R job:
```
* checking for file './DESCRIPTION' ... OK
* preparing 'RGF':
* checking DESCRIPTION meta-information ... OK
* installing the package to build vignettes
* creating vignettes ... OK
* checking for LF line-endings in source and make files and shell scripts
* checking for empty or unneeded directories
* building 'RGF_1.0.6.3.tar.gz'
1109
* using log directory 'C:/projects/rgf/R-package/RGF.Rcheck'
* using R version 3.6.2 (2019-12-12)
* using platform: x86_64-w64-mingw32 (64-bit)
* using session charset: ISO8859-1
* using option '--as-cran'
* checking for file 'RGF/DESCRIPTION' ... OK
* checking extension type ... Package
* this is package 'RGF' version '1.0.6.3'
* package encoding: UTF-8
* checking CRAN incoming feasibility ... NOTE
Maintainer: 'Lampros Mouselimis <mouselimislampros@gmail.com>'
1121
The Date field is over a month old.
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for executable files ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking serialization versions ... OK
* checking whether package 'RGF' can be installed ... OK
* checking installed package size ... OK
* checking package directory ... OK
* checking for future file timestamps ... OK
* checking 'build' directory ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... WARNING
Conversion of 'README.md' failed:
[WARNING] This document format requires a nonempty <title> element.
  Defaulting to 'README' as the title.
  To specify a title, use 'title' in metadata or --metadata title="...".
  Could not fetch https://codecov.io/gh/RGF-team/rgf/branch/master/graphs/badge.svg?branch=master
HttpExceptionRequest Request {
  host                 = "codecov.io"
  port                 = 443
  secure               = True
  requestHeaders       = []
  path                 = "/gh/RGF-team/rgf/branch/master/graphs/badge.svg"
  queryString          = "?branch=master"
  method               = "GET"
  proxy                = Nothing
  rawBody              = False
  redirectCount        = 10
  responseTimeout      = ResponseTimeoutDefault
  requestVersion       = HTTP/1.1
  }
 (InternalException (HandshakeFailed (Error_Protocol ("certificate has unknown CA",True,UnknownCa))))
 * checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking R files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking use of S3 registration ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd line widths ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking installed files from 'inst/doc' ... OK
* checking files in 'vignettes' ... OK
* checking examples ... OK
* checking for unstated dependencies in 'tests' ... OK
* checking tests ...
  Running 'testthat.R' [21s]
 OK
 * checking for unstated dependencies in vignettes ... OK
* checking package vignettes in 'inst/doc' ... OK
* checking re-building of vignette outputs ... OK
* checking PDF version of manual ... OK
* checking for detritus in the temp directory ... OK
* DONE

Status: 1 WARNING, 1 NOTE
```

Possible related: https://stackoverflow.com/a/61273758.

@mlampros Can you please help bring this badge back?

Reply:
Hi, I think this error appears currently in some R packages ( if you search on the web for  "Could not fetch https://codecov.io" ) but I'm not sure what the cause is. The weblink seems to be correct because it shows the code coverage on my webbrowser (https://codecov.io/gh/RGF-team/rgf/branch/master/graphs/badge.svg?branch=master)

Reply:
Also it is quite strange that the error appears only on Windows.

Reply:
OK, I think we can remove the badge at CI side only before performing any checks: e2bfa1a9ee8cc07d090ea25a2cd40b19faafd720.